.github/workflows/main.yml

```diff
@@ -3,8 +3,6 @@ name: main
 on:
   push:
     branches:
-      - main
-      - "[0-9]+.[0-9]+.x"
```

Comment:
this is intentional to prevent double-testing branches -- my recommendation is to instead add another prefix [such as what I do here](https://github.com/pre-commit/pre-commit/blob/e97140e4d082fb029820eee4fe9e649efc8b2121/azure-pipelines.yml#L3)

Reply:
I agree, this would be rather wasteful - instead of running CI twice for each change (PR + main), it will be run 3 times (branch + PR + main). I think that for local testing, @asottile's suggestion or a local PR would do.

Reply:
Sounds good, updated with `test-me-*`!